pysmt/test/test_qe.py

```diff
@@ -96,7 +143,7 @@ def _int_example(self, qe):
         f = ForAll([p], Implies(LT(Int(0), p), LT(q, p)))
         qf = qe.eliminate_quantifiers(f).simplify()
 
-        self.assertTrue(is_valid(Iff(qf, LE(q, Int(0)))))
+        self.assertTrue(is_valid(Iff(qf, LE(q, Int(0))), logic=QF_LIA))
```

Comment:
Why did you need to add this? If no logic is specified, it should be autodetected.
Maybe this is a symptom of a bug in the oracle?